project_national/panel_upgrades.yml

```diff
@@ -0,0 +1,1320 @@
+
+  - upgrade_name: 1.4 High Efficiency Space Heating Electrification 
+    options:
+    #### 1.0 Common Options For Heat Pump, 12 options ####
+      - *1_heating_fuel
+      - *2_cooling_efficiency_ducted
+      - *3_cooling_efficiency_ductless
+      - *4_100_percent_conditioned
+      - *6_cooling_setpoint_offset_magnitude_0
+      - *7_cooling_setpoint_offset_period_none
+      - *9_heating_setpoint_offset_magnitude_0
+      - *10_heating_setpoint_offset_period_none
+    #### 1.4 High-Efficiency Heat Pump With Electric Backup, 2 options  ####
+      - *1_high_eff_ashp_ducted
+      - *2_high_eff_mshp_ductless
+    #### 2.1 Electric Water Heating, 4 options ####
+      - *1_water_heater_fuel 
+      - *2_water_heater_fuel_eff_premium
+      - *3_water_heater_fuel_eff_standard
+      - *4_water_heater_fuel_eff_tankless
+    #### 3 Electric Resistance Cooking, 1 option  ####
+      - *electric_resistance_cooking
+    #### 4 Electric Clothes Dryer, 1 options  ####
+      - *clothes_dryer_electric 
+    #### 5 Electric Pool Heater And Hot Tub Spa Heater, 2 options  ####
+      - *1_electric_pool_heaters 
+      - *2_electric_spa_heaters
+
+  - upgrade_name: 1.5 High Efficiency Space Heating Electrification With HPWH  
+    options:
+    #### 1.0 Common Options For Heat Pump, 12 options ####
+      - *1_heating_fuel
+      - *2_cooling_efficiency_ducted
+      - *3_cooling_efficiency_ductless
+      - *4_100_percent_conditioned
+      - *6_cooling_setpoint_offset_magnitude_0
+      - *7_cooling_setpoint_offset_period_none
+      - *9_heating_setpoint_offset_magnitude_0
+      - *10_heating_setpoint_offset_period_none
+    #### 1.4 High-Efficiency Heat Pump With Electric Backup, 2 options  ####
+      - *1_high_eff_ashp_ducted
+      - *2_high_eff_mshp_ductless
+    #### 2.2 240V HPWH, 4 options  ####
+      - *1_water_heater_fuel_240V_hpwh
+      - *2_240V_hpwh_50_gal
+      - *3_240V_hpwh_66_gal
+      - *4_240V_hpwh_80_gal
+    #### 3 Electric Resistance Cooking, 1 option  ####
+      - *electric_resistance_cooking
+    #### 4 Electric Clothes Dryer, 1 options  ####
+      - *clothes_dryer_electric 
+    #### 5 Electric Pool Heater And Hot Tub Spa Heater, 2 options  ####
+      - *1_electric_pool_heaters 
+      - *2_electric_spa_heaters
+
+  - upgrade_name: 1.6. Low-voltage Electrification 
+    options:
+    #### 1.0 Common Options For Heat Pump, 12 options ####
+      - *1_heating_fuel
+      - *2_cooling_efficiency_ducted
+      - *3_cooling_efficiency_ductless
+      - *4_100_percent_conditioned
+      - *6_cooling_setpoint_offset_magnitude_0
+      - *7_cooling_setpoint_offset_period_none
+      - *9_heating_setpoint_offset_magnitude_0
+      - *10_heating_setpoint_offset_period_none
+    #### 1.2 Federal Minimum Heat Pump With Electric Backup, 2 options  ####
+      - *1_fed_min_ashp_ducted
+      - *2_fed_min_mshp_ductless
+    #### 2.1 Electric Water Heating, 4 options #### ##TO DO evise it to 120V HPWH
```

Comment:
TO BE UPDATED. If you are testing the run before this is updated, I'd suggest just commenting out the entire upgrade package.